docs_nnx/guides/gemma.ipynb

```diff
@@ -235,7 +261,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You're ready to start sampling ! This sampler uses just-in-time compilation, so changing the input shape triggers recompilation, which can slow things down. For the fastest and most efficient results, keep your batch size consistent."
```

Comment:
Added some background on JAX JIT after studying the source code (it's not NNX JIT).

"**Note:** This Flax NNX [`gemma.Sampler`](https://github.com/google/flax/blob/main/examples/gemma/sampler.py) uses JAX’s [just-in-time (JIT) compilation](https://jax.readthedocs.io/en/latest/jit-compilation.html), so changing the input shape triggers recompilation, which can slow things down. For the fastest and most efficient results, keep your batch size consistent."